src/test/java/starter/ReportSender.java

```diff
@@ -0,0 +1,50 @@
+package starter;
+
+import integrations.testrail.APIException;
+import integrations.testrail.TestRailManager;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
+import utils.ContextKeeper;
+
+import java.io.IOException;
+
+import static integrations.SlackIntegration.sendMessageToSlack;
+
+public class ReportSender {
+
+    protected String testCaseId;
+
+    @Before
+    public static void setPreconditions() {
+
+    }
+
+    @After
```

Comment:
I'm not sure you need to use cucumber After because the hook will run after each Scenario